v2/yarpchttp/handler.go

```diff
@@ -52,193 +49,197 @@ type handler struct {
 	logger              *zap.Logger
 }
 
-func (h handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+func (h handler) ServeHTTP(w http.ResponseWriter, hRequest *http.Request) {
 	responseWriter := newResponseWriter(w)
-	service := popHeader(req.Header, ServiceHeader)
-	procedure := popHeader(req.Header, ProcedureHeader)
-	bothResponseError := popHeader(req.Header, AcceptsBothResponseErrorHeader) == AcceptTrue
+	service := popHeader(hRequest.Header, ServiceHeader)
+	procedure := popHeader(hRequest.Header, ProcedureHeader)
+	bothResponseError := popHeader(hRequest.Header, AcceptsBothResponseErrorHeader) == AcceptTrue
```

Comment:
Let’s stick with req/res for the YARPC type throughout. We can distinguish the HTTP as httpReq for a good balance of explicitness and brevity.